src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/DefaultRazorLanguageServerCustomMessageTarget.cs

```diff
@@ -413,6 +413,7 @@ await foreach (var response in requests)
             ProvideSemanticTokensRangeParams semanticTokensParams,
             CancellationToken cancellationToken)
         {
+            // We're not actually serializing the Range correctly here. Likely we need to define a jsonconverter for MS.VS.Range
```

Comment:
Is there a work item for this? Also, out of curiosity what part isn't being serialized correctly?

Reply:
Oh, this is no longer true. This was me leaving myself a note to remember my current thinking across days and forgetting to clean it up. Will clean it up.